index.html

```diff
@@ -0,0 +1,220 @@
+                <ul class="nav-menu">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+                <div class="humburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>       
+        </header>
+        <main class="main">                       
+            <section class="section1">
+                <h1 class="greeting-text">Hey there. I’m <br> Alijan.</h1>
+                <h2 class="profission-text">I’m a software developer</h2>
+                <p class="supporting-text">
+                    I can help you build a product,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                
+                <div class="social-media">
+                    <ul>
+                        <li>
+                            <a href="https://github.com/rahimialijan">
+                                <img src="images/Icons/Icon-GitHub.svg" alt="GitHub">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/alijan-rahimi-18389ab3/">
+                                <img src="images/Icons/Icon-Linkedin.svg" alt="Linkedin">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                <img src="images/Icons/Icon-Angelist.svg" alt="Angelist">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/AlijanRahimi10">
+                                <img src="images/Icons/Icon-Twitter.svg" alt="Twitter">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                <img src="images/Icons/Icon-Medium.svg" alt="Medium">
+                            </a> 
+                        </li>
+                    </ul>
+                </div>
+                <div class="action-arrow-down">
+                    <a href="#recent-work">
+                        <img src="images/Icons/Icon-Arrow-button.svg" alt="action button arrow down">
+                    </a>
+                </div>
+            </section> 
+             <!------------------------------------Works Section------------------------------------>
+            <section class="work-section">
+                <div class="works-title">
+                    <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly use `h2` or `h3` tag here to get better accessibility of the website.
- As you have used `h1` tag for the main title of this site at `section1` so you need to use other `hx` tags for the subtitle of this site to maintain the hierarchy level.
- Example:
  - `<h1> My name </h1>` ----> Title
    - `<h2> My study </h2>`  ----> Subtitle
      - `<h3> My birthdate </h3>` ----> Subtitle

Reply:
Hi Dear @Tufahel, I changed the h1 to the h2 tag, thanks for your guidance.